lib/Listener/FirstLoginListener.php

```diff
@@ -62,6 +61,9 @@ public function __construct(IMailer $mailer, NotificationTrackerService $service
 	 * @throws NotFoundException
 	 */
 	public function handle(IUser $user): void {
+		if (!$enabled) {
+			return; // Feature is disabled, cancel sending initial login email
```

Comment:
`$trackedNotification = $this->service->find($user->getUID());` should still be called i think in order to initialize the database for the user to send out the monthly mail later on

Reply:
yes, good catch.